BookingsApi/BookingsApi.DAL/Commands/RestoreJusticeUserCommand.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Exceptions;
+using Microsoft.EntityFrameworkCore;
+
+namespace BookingsApi.DAL.Commands
+{
+    public class RestoreJusticeUserCommand : ICommand
+    {
+        public Guid Id { get; }
+
+        public RestoreJusticeUserCommand(Guid id)
+        {
+            Id = id;
+        }
+    }
+    
+    public class RestoreJusticeUserCommandHandler : ICommandHandler<RestoreJusticeUserCommand>
+    {
+        private readonly BookingsDbContext _context;
+
+        public RestoreJusticeUserCommandHandler(BookingsDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task Handle(RestoreJusticeUserCommand command)
+        {
+            var justiceUser = await _context.JusticeUsers
+                .IgnoreQueryFilters()
```

Comment:
With the amount of ignoring, I would argue to get rid of this default filter.

Reply:
I'm not so sure, this is really the only place we want to interact with deleted users. We could append `where !isDeleted` onto all of our queries and usages, but it will be easy to forget to do so when adding new functionality